src/api/v3/events.py

```diff
@@ -0,0 +1,95 @@
+from django.http import JsonResponse
+
+from backend.settings import WEBAPP_URL, TIME_ZONE
```

Comment:
```suggestion
from backend.settings import WEBAPP_URL, CURRENT_TIME_ZONE
```
`TIMEZONE` is the timezone of our server, which will always be `UTC`. The setting we're looking for is `CURRENT_TIME_ZONE`, which is the timezone we expect our users to be in (and thus also the events which are created by them). Maybe the event model will be extended with it's own timezone setting in the future (see #663), but until then, I sugges to use `CURRENT_TIME_ZONE`.